pants.ini

```diff
@@ -176,6 +166,15 @@ no_warning_args: [
     '-S-nowarn',
   ]
 
+[lint.checkstyle]
```

Comment:
I'd rather kill the migrate_config script. The config verifier does this better, and uses information that is already used for deprecation.